types/subtext/index.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'subtext' {
```

Comment:
`@hapi/subtext` has types, but not `subtext`. This should do until apollo-server upgrades to the `@hapi`-scoped packages.

Reply:
I'm wrong, there aren't official types: https://www.npmjs.com/package/@types/hapi__subtext. Perhaps I should contribute these back to the `@types` repo.